src/tablist.js

```diff
@@ -65,12 +71,18 @@ SideTabList.prototype = {
     document.addEventListener("dragover", e => this.onDragOver(e));
     document.addEventListener("drop", e => this.onDrop(e));
 
+    // Scroll to switch tabs
+    document.addEventListener("wheel", e => this.onScroll(e));
```

Comment:
Sure, that shouldn't be a problem. I'll add that in the next few days/next week, when I have more free time.